src/Compilers/Core/Portable/Diagnostic/FileLinePositionSpan.cs

```diff
@@ -20,21 +22,24 @@ namespace Microsoft.CodeAnalysis
         /// <remarks>
         /// Path may be <see cref="string.Empty"/> if not available.
         /// </remarks>
+        [DataMember(Order = 0)]
         public string Path { get; }
 
+        /// <summary>
+        /// Gets the span.
+        /// </summary>
+        [DataMember(Order = 1)]
+        public LinePositionSpan Span { get; }
```

Comment:
LinePositionSpan was already made into a DataContract.  But we don't have hte equiv for FileLinePositionSpan yet.

Reply:
@dotnet/roslyn-compiler  for this trivial change.  This is needed so we can use this primitive type in our OOP serialization calls.